pyInterface/package/_likelihood.py

```diff
@@ -1,65 +1,93 @@
 import pyRootPwa
 ROOT = pyRootPwa.ROOT
 
-def initLikelihood(waveDescThres,
-                   massBinCenter,
-                   ampFileList,
-                   normIntegralFileName,
-                   accIntegralFileName,
-                   accEventsOverride = 0,
-                   cauchy = False,
-                   cauchyWidth = 0.5,
-                   rank = 1,
-                   verbose = False
-                  ):
-	likelihood = pyRootPwa.core.pwaLikelihood()
-	likelihood.useNormalizedAmps(True)
-	if (not verbose):
-		likelihood.setQuiet()
-	if cauchy:
-		likelihood.setPriorType(pyRootPwa.core.HALF_CAUCHY)
-		likelihood.setCauchyWidth(cauchyWidth)
-	if (not likelihood.init(waveDescThres,
-	                        rank,
-	                        massBinCenter)):
-		printErr("could not initialize likelihood. Aborting...")
+def initLikelihood(ampFileList, 
+                   normIntegralFileName, 
+                   accIntegralFileName, 
+                   binningMap, 
+                   waveDescThres, 
+                   cauchy            = False, 
+                   cauchyWidth       = 0.5,  
+                   accEventsOverride = 0, 
+                   rank              = 1, 
+                   verbose           = False, 
+                   eventFileNameList   = []
+                   ):
+	massBinCenter = float(binningMap['mass'][1] + binningMap['mass'][0]) / 2. /1000 # YOU CAN DO BETTER
+ 
+ 	likelihood = pyRootPwa.core.pwaLikelihood()
+ 	likelihood.useNormalizedAmps(True)
+ 	if (not verbose):
+ 		likelihood.setQuiet()
+ 	if cauchy:
+ 		likelihood.setPriorType(pyRootPwa.core.HALF_CAUCHY)
+ 		likelihood.setCauchyWidth(cauchyWidth)
+ 	if (not likelihood.init(waveDescThres,
+ 	                        rank,
+ 	                        massBinCenter)):
+		pyRootPwa.utils.printErr("could not initialize likelihood. Aborting...")
 		return None
-
-	normIntFile = ROOT.TFile.Open(normIntegralFileName, "READ")
-	if len(normIntFile.GetListOfKeys()) != 1:
-		pyRootPwa.utils.printWarn("'" + normIntegralFileName + "' does not contain exactly one TKey.")
+ 
+ 	normIntFile = ROOT.TFile.Open(normIntegralFileName, "READ")
+ 	if len(normIntFile.GetListOfKeys()) != 1:
+ 		pyRootPwa.utils.printWarn("'" + normIntegralFileName + "' does not contain exactly one TKey.")
```

Comment:
Could you fix the indentation? There are extra spaces at the beginning of the lines (does the python interpreter even allow this?)...